en/deploy/README.md

```diff
@@ -130,159 +130,118 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
 
-This will pull down a copy of your code onto PythonAnywhere. Check it out by typing `tree my-first-blog`:
+Deploying a web app on PythonAnywhere involves pulling down your code from GitHub, and then configuring PythonAnywhere to recognise it and start serving it as a web application.  There are manual ways of doing it, but PythonAnywhere provide a helper tool that will do it all for you. Let's install it first:
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ tree my-first-blog
-my-first-blog/
-├── blog
-│   ├── __init__.py
-│   ├── admin.py
-│   ├── migrations
-│   │   ├── 0001_initial.py
-│   │   └── __init__.py
-│   ├── models.py
-│   ├── tests.py
-│   └── views.py
-├── manage.py
-└── mysite
-    ├── __init__.py
-    ├── settings.py
-    ├── urls.py
-    └── wsgi.py
+$ pip3.6 install --user pythonanywhere
 ```
 
+That should print out some things like `Collecting pythonanywhere`, and eventually end with a line saying `Successfully installed (...) pythonanywhere- (...)`.
 
-### Creating a virtualenv on PythonAnywhere
-
-Just like you did on your own computer, you can create a virtualenv on PythonAnywhere. In the Bash console, type:
+Now we run the helper to automatically configure our app from GitHub. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd my-first-blog
-
-$ virtualenv --python=python3.6 myvenv
-Running virtualenv with interpreter /usr/bin/python3.6
-[...]
-Installing setuptools, pip...done.
-
-$ source myvenv/bin/activate
-
-(myvenv) $  pip install django~=1.11.0
-Collecting django
-[...]
-Successfully installed django-1.11.3
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
 ```
 
+As you watch that running, you'll be able to see what it's doing:
 
-> **Note** The `pip install` step can take a couple of minutes.  Patience, patience!  But if it takes more than five minutes, something is wrong.  Ask your coach.
+- Downloading your code from GitHub
+- Creating a virtualenv on PythonAnywhere, just like the one on your own PC
+- Updating your settings file with some deployment settings
+- Setting up a database on PythonAnywhere using the "migrate" command
+- Setting up your static files (we'll learn about these later)
+- And configuring PythonAnywhere to serve your web app via its API
 
-<!--TODO: think about using requirements.txt instead of pip install.-->
+On PythonAnywhere all those steps are automated, but they're the same steps you
+would have to go through with any other server provider.  The main thing to notice
+right now is that your database on PythonAnywhere is actually totally separate from
+your database on your own PC -- that means it can have different posts and admin accounts.
```

Comment:
Will Git**Book** show this `--` as an mdash (&mdash;) or ndash (&ndash;) or so, instead of a double-minus/dash? (The [Git**Hub** preview](https://github.com/hjwp/djangogirls-tutorial/blob/5378bbcd4c10e2a20788605c49cd5694ebb57add/en/deploy/README.md) shows it as the latter.)

Reply:
I'll replace it with an explict mdash literal just in case.